Formula/at-spi2-atk.rb

```diff
@@ -84,8 +84,9 @@ def install
       }
     EOS
 
-    pkg_config_cflags = shell_output("pkg-config --cflags --libs atk-bridge-2.0 glib-2.0 atk").chomp.split
-    system ENV.cc, "test.c", *pkg_config_cflags, "-o", "test"
+    pkg_config_cmd = Formula["pkg-config"].opt_bin/"pkg-config --cflags --libs atk-bridge-2.0 glib-2.0 atk"
```

Comment:
We don't need to specify the full path to `pkg-config`. We can just use the one in `PATH`. We only need to specify the path for executables provided by the formula.

Reply:
we have mixed usage of pkg-config in the repo (some do specify the pkg-path, some do not)

Reply:
But it doesn't matter except in the pkg-config formula. We only need to ensure that we're not testing the system version of our packages.